pkg/tests/api/alerting/api_prometheus_test.go

```diff
@@ -147,7 +148,7 @@ func TestPrometheusRules(t *testing.T) {
 			"rules": [{
 				"state": "inactive",
 				"name": "AlwaysFiring",
-				"query": "[{\"datasourceUid\":\"-100\",\"expression\":\"2 + 3 \\u003e 1\",\"intervalMs\":1000,\"maxDataPoints\":100,\"type\":\"math\"}]",
+				"query": "[{\"refId\":\"A\",\"queryType\":\"\",\"relativeTimeRange\":{\"from\":18000,\"to\":10800},\"datasourceUid\":\"\",\"model\":{\"datasourceUid\":\"-100\",\"expression\":\"2 + 3 \\u003e 1\",\"intervalMs\":1000,\"maxDataPoints\":100,\"type\":\"math\"}}]",
```

Comment:
I would suggest removing the `datasourceUid` from the model [here](https://github.com/grafana/grafana/blob/827e1e57b56ecb57843363710761dc32c4204d9f/pkg/tests/api/alerting/api_prometheus_test.go#L75) for conforming better with the actual case.